scipy/optimize/_optimize.py

```diff
@@ -1314,7 +1317,7 @@ def fmin_bfgs(f, x0, fprime=None, args=(), gtol=1e-5, norm=Inf,
 def _minimize_bfgs(fun, x0, args=(), jac=None, callback=None,
                    gtol=1e-5, norm=Inf, eps=_epsilon, maxiter=None,
                    disp=False, return_all=False, finite_diff_rel_step=None,
-                   **unknown_options):
+                   xtol=1e-6, **unknown_options):
```

Comment:
@WillTirone I'd suggest making the default 0 so that it doesn't change the behavior of existing code.
```suggestion
                   xtol=0, **unknown_options):
```